tests/integration/components/editor-with-preview-test.js

```diff
@@ -1,8 +1,33 @@
+import Ember from 'ember';
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 
+let mockPreview = Ember.Object.create({
+  markdown: 'A **body**',
+  body: 'A <strong>body</strong>',
+  save() {
+    return Ember.RSVP.resolve(this);
+  }
+});
```

Comment:
I still love that this is one of our examples. Just had to point that out. Oldie but goodie.